tests/test_audio_start_end_time.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <nopemd.h>
+
+int main(int ac, char **av)
+{
+    if (ac < 2) {
+        fprintf(stderr, "Usage: %s <media.mkv> [<use_pkt_duration>]\n", av[0]);
+        return -1;
+    }
+
+    const char *filename = av[1];
+    const int use_pkt_duration = ac > 2 ? atoi(av[2]) : 0;
+
+    int i = 0, ret = 0, smp = 0;
+    struct nmd_ctx *s = nmd_create(filename);
+
+    if (!s)
+        return -1;
+
+    nmd_set_option(s, "auto_hwaccel", 0);
+    nmd_set_option(s, "use_pkt_duration", use_pkt_duration);
+    nmd_set_option(s, "avselect", NMD_SELECT_AUDIO);
+    nmd_set_option(s, "audio_texture", 0);
+    nmd_set_option(s, "end_time", 30.0);
+    nmd_set_option(s, "end_time", 90.0);
```

Comment:
Why the dup? Did you meant `start` for the first one?

Reply:
Fixed.